packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -0,0 +1,1163 @@
+/*
```

Comment:
Does this multi addresses shipping work for guest user? or only registered users?

Reply:
In this PR it is only for Registered User with atleast one Address in Address Book and more than one items in cart.